README.md

```diff
@@ -30,7 +36,8 @@ The Secrets Store CSI driver `secrets-store.csi.k8s.io` allows Kubernetes to mou
   - [Unit Tests](#unit-tests)
   - [End-to-end Tests](#end-to-end-tests)
 - [Known Issues and Workarounds](#known-issues-and-workarounds)
-- [Contributing](#contributing)
+- [Troubleshooting](#troubleshooting)
```

Comment:
While you are in here, pls remove `Known Issues and Workarounds` altogether

Reply:
Updated!